forc-plugins/forc-doc/src/cli.rs

```diff
@@ -22,8 +22,8 @@ forc_util::cli_examples! {
 pub struct Command {
     /// Path to the Forc.toml file. By default, forc-doc searches for the Forc.toml
```

Comment:
Let's also align the description with the `forc build` and `forc fmt`. Here I need to notice that those two tools already have a slightly different description. The proposal is to take the description from the `forc build` and use it also for `forc fmt` and `forc docs`. The proposal is to do this slight change in `forc fmt` also as a part of this PR.